docs/tutorials/enterprise_beginner/1_get_a_starter_website.md

```diff
@@ -58,40 +58,58 @@ Edit it to remove the Image Field that has a Content Relation (ezobjectrelation)
 
-Place the [`pagelayout.html.twig`](https://github.com/ezsystems/ezstudio-beginner-tutorial/blob/v2-step1/app/Resources/views/pagelayout.html.twig) and [`pagelayout_menu.html.twig`](https://github.com/ezsystems/ezstudio-beginner-tutorial/blob/v2-step1/app/Resources/views/pagelayout_menu.html.twig) files in `app/Resources/views` folder. Create a new folder, called `full`, under `views`. Place further template files in it:
+Place the [`pagelayout.html.twig`](https://github.com/ezsystems/ezplatform-ee-beginner-tutorial/blob/v3-step1/templates/pagelayout.html.twig) and [`pagelayout_menu.html.twig`](https://github.com/ezsystems/ezplatform-ee-beginner-tutorial/blob/v3-step1/templates/pagelayout_menu.html.twig) files in the `templates` folder. Create a new folder, called `full`, under `templates`. Place further template files in it:
 
-- [`article.html.twig`](https://github.com/ezsystems/ezstudio-beginner-tutorial/blob/v2-step1/app/Resources/views/full/article.html.twig)
-- [`dog_breed.html.twig`](https://github.com/ezsystems/ezstudio-beginner-tutorial/blob/v2-step1/app/Resources/views/full/dog_breed.html.twig)
-- [`folder.html.twig`](https://github.com/ezsystems/ezstudio-beginner-tutorial/blob/v2-step1/app/Resources/views/full/folder.html.twig)
-- [`tip.html.twig`](https://github.com/ezsystems/ezstudio-beginner-tutorial/blob/v2-step1/app/Resources/views/full/tip.html.twig)
+- [`article.html.twig`](https://github.com/ezsystems/ezplatform-ee-beginner-tutorial/blob/v3-step1/templates/full/article.html.twig)
+- [`dog_breed.html.twig`](https://github.com/ezsystems/ezplatform-ee-beginner-tutorial/blob/v3-step1/templates/full/dog_breed.html.twig)
+- [`folder.html.twig`](https://github.com/ezsystems/ezplatform-ee-beginner-tutorial/blob/v3-step1/templates/full/folder.html.twig)
+- [`tip.html.twig`](https://github.com/ezsystems/ezplatform-ee-beginner-tutorial/blob/v3-step1/templates/full/tip.html.twig)
 
-Place two configuration files in `app/config` folder:
+Place two configuration files in the `config` folder:
 
-- [`views.yaml`](https://github.com/ezsystems/ezstudio-beginner-tutorial/blob/v2-step1/app/config/views.yaml)
-- [`image_variations.yaml`](https://github.com/ezsystems/ezstudio-beginner-tutorial/blob/v2-step1/app/config/image_variations.yaml)
+- [`views.yaml`](https://github.com/ezsystems/ezplatform-ee-beginner-tutorial/blob/v3-step1/config/views.yaml)
+- [`image_variations.yaml`](https://github.com/ezsystems/ezplatform-ee-beginner-tutorial/blob/v3-step1/config/image_variations.yaml)
 
-Modify the `config.yaml` file located in `app/config` folder and add the following lines at the end of the `imports` block:
+Modify the `ezplatform.yaml` file located in `config/packages` folder and add the following block at the start of the file:
 
 ``` yaml
-    - { resource: views.yaml }
-    - { resource: image_variations.yaml }
+imports:
```

Comment:
Apparently, the additional config files created in `config/packages` will load automatically. We can keep suggesting to add `imports` commands or drop it — to be discussed (the same applies to eZ Platform tutorial). 😉